libraries/Microsoft.Bot.Builder/BotAdapter.cs

```diff
@@ -282,20 +282,6 @@ protected async Task RunPipelineAsync(ITurnContext turnContext, BotCallbackHandl
             // Call any registered Middleware Components looking for ReceiveActivityAsync()
             if (turnContext.Activity != null)
             {
-                if (turnContext.Activity.Locale != null)
-                {
-                    try
-                    {
-                        Thread.CurrentThread.CurrentCulture = new CultureInfo(turnContext.Activity.Locale);
-                        (turnContext as TurnContext).Locale = turnContext.Activity.Locale;
```

Comment:
This code was pretty dangerous because it broke things for folks who implemented their own `ITurnContext`. It's generally good to not have it anymore.